packages/apollo-gateway/src/__tests__/build-query-plan.feature

```diff
@@ -683,7 +683,7 @@ Scenario: for abstract types, it should add __typename when fetching objects of
         "kind": "Fetch",
```

Comment:
@JakeDawkins at this point in time is it acceptable to be updating the `.feature` file like this? There should be no diff in the actual query plan itself, just in the ordering of some of the nodes.